lib/crowbar/client/app/backup.rb

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright 2015, SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Crowbar
+  module Client
+    module App
+      class Backup < Base
+        desc "list",
+          "List existing backups"
+
+        long_desc <<-LONGDESC
+          `list` will print out a list of existing backups on the admin
+          node. You can display the list in different output
+          formats and you can filter the list by any search criteria.
+          With --format <format> option you can choose an output format
+          with the available options table, json or plain. You can also
+          use the shortcut options --table, --json or --plain.
+          With --filter <filter> option you can limit the result of
```

Comment:
Add an empty line, see it at https://github.com/crowbar/crowbar-client/blob/master/lib/crowbar/client/app/barclamp.rb#L32